tests/e2e/statefulsets.go

```diff
@@ -93,6 +100,32 @@ var _ = ginkgo.Describe("statefulset", func() {
 		if err == nil && service != nil {
 			deleteService(namespace, client, service)
 		}
+		if stretchedSVC {
+			zonalPolicy = GetAndExpectStringEnvVar(envZonalStoragePolicyName)
+			labels_ns = map[string]string{}
+			labels_ns[admissionapi.EnforceLevelLabel] = string(admissionapi.LevelPrivileged)
+			labels_ns["e2e-framework"] = f.BaseName
+
+			if zonalPolicy == "" {
+				ginkgo.Fail(envZonalStoragePolicyName + " env variable not set")
+			}
+			zonalWffcPolicy = GetAndExpectStringEnvVar(envZonalWffcStoragePolicyName)
+			if zonalWffcPolicy == "" {
+				ginkgo.Fail(envZonalWffcStoragePolicyName + " env variable not set")
+			}
+			framework.Logf("zonal policy: %s and zonal wffc policy: %s", zonalPolicy, zonalWffcPolicy)
+
+			topologyHaMap := GetAndExpectStringEnvVar(topologyHaMap)
+			_, categories = createTopologyMapLevel5(topologyHaMap)
```

Comment:
I think this code has changed, we need to use only createAllowedTopolgies method, refer https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/tests/e2e/no_hci_mesh_rwx_multivc_topology.go#L101C23-L101C45

Reply:
Please use same method across all the files to maintain uniformity